typescript/infra/config/environments/testnet4/gas-oracle.ts

```diff
@@ -29,6 +29,7 @@ const gasPrices: ChainMap<BigNumber> = {
   chiado: ethers.utils.parseUnits('2', 'gwei'),
   solanatestnet: ethers.BigNumber.from('28'),
   eclipsetestnet: ethers.BigNumber.from('28'),
+  plumetestnet: ethers.utils.parseUnits('1.5', 'gwei'),
```

Comment:
this is quite different from the override you are using?

Reply:
Yaaa, this was a 150000% premium, I've dropped it to 0.01 gwei which is 1000% premium